lib/fluent/plugin/out_sumologic.rb

```diff
@@ -229,6 +255,11 @@ def write(chunk)
             record = { @timestamp_key => sumo_timestamp(time) }.merge(record)
           end
           log = dump_log(merge_json(record))
+        when 'fields'
+          if @add_timestamp
+            record = { :timestamp => sumo_timestamp(time) }.merge(record)
```

Comment:
Could we use `@timestamp_key` here as well?

Reply:
good catch, will fix